src/components/events/event-carousel.svelte

```diff
@@ -1,13 +1,52 @@
 <script lang="ts">
+  import { onMount } from "svelte";
   import EventItem from "./event-item.svelte";
   import type { AcmEvent } from "../../lib/parse-ical-data";
 
+  enum GrabbingState {

```

Comment:
Do we need this? Why can't we do `grabbing bool`?

Reply:
I am using this enum because it is being used as an attribute value. I assume there is a simpler way to do this involving a boolean. In the end, the CSS just needs to target something to be able to give it the appropriate style which could be a class or an attribute.

We could possibly instead use an `isGrabbing` variable and use it to assign a class onto the div using a Svelte class directive like this: `<div class:grabbing={isGrabbing} ... />`.

Reply:
Addressed in commit <https://github.com/EthanThatOneKid/acmcsuf.com/pull/91/commits/b2d9ffd220775905eb63197f599820092cbffe6f>.